ui/src/app/workflows/components/workflows-row/workflows-row-name.tsx

```diff
@@ -0,0 +1,18 @@
+import * as kubernetes from 'argo-ui/src/models/kubernetes';
+import * as React from 'react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+import {ANNOTATION_DESCRIPTION, ANNOTATION_TITLE} from '../../../shared/annotations';
+
+require('./workflows-row.scss');
+
+export const WorkflowsRowName = ({metadata}: {metadata: kubernetes.ObjectMeta}) => {
+    const title = (metadata.annotations && metadata.annotations[ANNOTATION_TITLE] && metadata.annotations[ANNOTATION_TITLE]) || metadata.name;
```

Comment:
is this a typo the double use of `metadata.annotations[ANNOTATION_TITLE]` or on purpose?

Reply:
Typo. Great catch. Will fix.